lib/widget/dropdown.dart

```diff
@@ -141,7 +141,7 @@ abstract class SelectOne extends StatefulWidget with Invokable, HasController<Se
 
   void calculateIconSize(List v) {
     List iconSize = [];
-    _controller.size = 0;
```

Comment:
This is not what I meant. This variable is the 'iconWidth' so it should be named similar to that.

Reply:
List created for icon size is for calculating maximum size and if i do this directly then function will render